gsk4/src/gl_shader.rs

```diff
@@ -0,0 +1,61 @@
+use glib::translate::*;
```

Comment:
This file needs to be used in `src/lib.rs`.  it also doesn't compile, you need at least a `use crate::GLShader`